app/models/goal.py

```diff
@@ -3,3 +3,13 @@
 
```

Comment:
Great job completing Task List Angela! Your code looks clean and easily readable. I want to point out the good variable naming and your code is DRY. Great use of helper methods in your models! Excellent work using blue prints & creating RESTful CRUD routes for each model.